### PR TITLE
Adding my keymap for 2% Milk keyboard

### PR DESCRIPTION
**Pull request by encg**
## Description

This keymap brings a "slots" game to the 2% Milk keyboard. Top button outputs all winning combos and bottom button is a "spin." More

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* N/A

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/spaceman/2_milk/keymaps/encg/config.h [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#define UNICODE_SELECTED_MODES UC_WINC, UC_WIN, UC_MAC, UC_LNX
```
Comment (zvecr) [inline keyboards/spaceman/2_milk/keymaps/encg/config.h:1]:
```suggestion
#pragma once

#define UNICODE_SELECTED_MODES UC_WINC, UC_WIN, UC_MAC, UC_LNX
```

**Review by zvecr — CHANGES_REQUESTED** [review]
Also gpl compatible licence headers need to be added to *.c and *.h files

File: keyboards/spaceman/2_milk/keymaps/encg/config.h (hunk 1 above)
Comment (encg) [reply keyboards/spaceman/2_milk/keymaps/encg/config.h]: This change was made in latest commit along with the addition of GPL license headers

**Comment by encg** [comment]
Added GPL license headers and made the requested change. Also realized I should not have committed to my `master` .

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**